ScrumPoker.DataBase.Interfaces/IGameRoomData.cs

```diff
@@ -0,0 +1,12 @@
+using ScrumPoker.Core.Models;
+
+namespace ScrumPoker.DataBase.Interfaces;
+
+public interface IGameRoomData
```

Comment:
Interfaces should have xml comments.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags
If using `Rider` putting three slashes `///` before method/class name should generate boilerplate documentation
Also add `inheritdoc` to implementation to inherit comments from interface